src/Symfony/Component/Validator/Resources/translations/validators.th.xlf

```diff
@@ -298,6 +298,74 @@
                 <source>The image is portrait oriented ({{ width }}x{{ height }}px). Portrait oriented images are not allowed.</source>
                 <target>ภาพนี้เป็นแนวตั้ง ({{ width }}x{{ height }}px) ไม่อนุญาตภาพที่เป็นแนวตั้ง</target>
             </trans-unit>
+            <trans-unit id="78">
+                <source>An empty file is not allowed.</source>
+                <target>ไม่อนุญาตให้ใช้ไฟล์ว่าง</target>
+            </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>ไม่สามารถแก้ไขชื่อโฮสต์</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>ค่านี้ไม่ตรงกับการเข้ารหัส {{charset}}</target>
+            </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>นี่ไม่ถูกต้องตามรหัสสำหรับระบุธุรกิจนี้ (BIC)</target>
+            </trans-unit>
+            <trans-unit id="82">
+                <source>Error</source>
+                <target>เกิดข้อผิดพลาด</target>
+            </trans-unit>
+            <trans-unit id="83">
+                <source>This is not a valid UUID.</source>
+                <target>นี่ไม่ใช่ UUID ที่ถูกต้อง</target>
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>ค่านี้ควรเป็น {{compar_value}} หลายตัว</target>
```

Comment:
Please, add missing spaces after `{{` and before `}}`

Reply:
Thanks for your review on this, I also update missing space on id 80 and 85 and 94 too.